soweego/validator/checks.py

```diff
@@ -779,53 +970,9 @@ def _dump_csv_output(data, outpath, log_msg_subject):
         with open(outpath, 'w') as ids_out:
             writer = csv.writer(ids_out)
             writer.writerows(data)
-        LOGGER.info('%s to be added dumped to %s', log_msg_subject, outpath)
+        LOGGER.info('%s dumped to %s', log_msg_subject, outpath)
     else:
-        LOGGER.info("No %s to be added, won't dump to file", log_msg_subject)
-
-
-def _load_wd_cache(file_handle):
```

Comment:
any reason why you removed this function altogether?

Reply:
Because we switched to Pickle: `pickle.load` replaces the old function